docs/Velox.md

```diff
@@ -1,16 +1,17 @@
 # Velox Backend
 
-Currently the mvn script can automatically fetch and build all dependency libraries incluing Velox and Arrow. Our nightly build still use Velox under oap-project. 
+Currently, the mvn script can automatically fetch and build all dependency libraries incluing Velox and Arrow. Our nightly build still use Velox under oap-project. 
 
 ## Prerequisite
 
-Velox use the script setup-ubuntu.sh to install all dependency libraries, but Arrow's dependency library can't be installed. So we need to install them manually:
+Velox uses the script setup-ubuntu.sh to install all dependency libraries, but Arrow's dependency libraries can't be installed.
+Velox also requires ninja for compilation. So we need to install them manually:
 
 ```shell script
-apt install maven build-essential cmake libssl-dev libre2-dev libcurl4-openssl-dev clang lldb lld libz-dev git
+apt install maven build-essential cmake libssl-dev libre2-dev libcurl4-openssl-dev clang lldb lld libz-dev git ninja-build
 ```
 
-Also we need to setup the JAVA_HOME env.
+Also, we need to set up the JAVA_HOME env. Currently, java8 is required.
```

Comment:
maybe also mention java 11/17 support not ready?

Reply:
Added.